monkey/monkey_island/cc/services/reporting/report.py

```diff
@@ -178,32 +180,53 @@ def get_exploited():
 
     @staticmethod
     def get_stolen_creds():
-        PASS_TYPE_DICT = {'password': 'Clear Password', 'lm_hash': 'LM hash', 'ntlm_hash': 'NTLM hash'}
         creds = []
+
+        # stolen creds from system info collectors
         for telem in mongo.db.telemetry.find(
                 {'telem_category': 'system_info', 'data.credentials': {'$exists': True}},
                 {'data.credentials': 1, 'monkey_guid': 1}
         ):
```

Comment:
Ideally, this should be converted into DAL, same as other modules. That's a bit our of scope, so the best thing would be to convert this to a method and call that method to return all credentials (zerologon included)